docs/blog/2019-04-03-how-to-build-a-blog-with-wordpress-and-gatsby-part-1/index.md

```diff
@@ -0,0 +1,122 @@
+
+### ACF to REST API
+
+Another plugin that works hand-in-hand with ACF is called [ACF to REST API](https://github.com/airesvsg/acf-to-rest-api). I bet you already guessed what it does. It exposes the data from your custom fields to the REST API so you can use it in your Gatsby.js site.
+
+### WP API Menus
+
+One of the great functions in WordPress is the ability for users to easily create and update menus for their site. If you hardcode the menu into your Gatsby site, it removes this ability for your users. To make it flexible for them, you can install the [WP API Menus](https://github.com/unfulvio/wp-api-menus) plugin to create REST routes for the menu items. Please note: There are a few different plugins built to create APIs from menus. According to the [gatsby-source-wordpress docs](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress), it must be this particular plugin to work.
+
+### WP Trigger Netlify Build
+
+<video autoplay muted loop>
+  <source src="./images/wp_netlify_build.webm">
+  <source src="./images/wp_netlify_build.mp4">
+</video>
+
+When I build a Gatsby website, I like to use Netlify to handle the deployment of the site. Netlify rebuilds whenever a new commit is pushed or merged to the master branch of the repository, but what about when someone updates their WordPress site?
+
+Netlify has the ability to create webhooks that you can grab to say, "Hey, there's new content so you need to rebuild". I built [WP Trigger Netlify Build](https://github.com/iamtimsmith/wp-trigger-netlify-build) to make this super easy to integrate with WordPress. Simply drop in the necessary information and it will tell Netlify to rebuild when changes are made. It even shows a badge with the status of the build on the dashboard.
+
+<video autoplay muted loop>
+  <source src="https://media.giphy.com/media/qyX9oq2ZmsPwk/giphy.mp4">
+</video>
+
+## You have themes and plugins, now what?
+
+Not to be too anti-climactic, but there's really not a ton more to do on the WordPress side other than create content. These plugins and theme will pretty much expose everything you need to build your Gatsby.js blog.
+
+That being said, if you are creating custom content types for your blog you will need a way to differentiate them in ACF. Creating a custom page template will serve this purpose so you can filter on the page template to show the custom fields. you can also use the page template to filter in your Gatsby project to make sure you are receiving the desired fields for a given page.
+
+<br /><br />
```

Comment:
This necessary?

Reply:
@DSchau  It doesn't directly affect the build for the article, although I thought it may be useful information to have if the reader decides they need custom post types. If you'd like, I can remove it though.

Reply:
@DSchau Should I remove this piece or leave it in as extra information for future builds?

Reply:
I'm referring to the breaking tags? Is that what you're referring to?

Reply:
@DSchau Oh! I thought you were looking at the content. I'll get the breaking tags removed!

Reply:
@DSchau I have removed the breaking tags.